src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_transformers.py

```diff
@@ -61,3 +61,14 @@ def pricesheet_show_properties(result):
 def transform_pricesheet_show_output(result):
     result.pricesheets = [pricesheet_show_properties(item) for item in result.pricesheets]
     return result
+
+
+def marketplace_list_output(result):
+    result.pretax_cost = str(result.pretax_cost)
+    result.consumed_quantity = str(result.consumed_quantity)
+    result.resource_rate = str(result.resource_rate)
```

Comment:
We need it, as marketplace method in transformer supports Json serialization in running tests.

Reply:
Instead of having a custom output transformer, use the `type=str` kwarg when you register the arguments like this in the `params.py` file:
```Python
c.argument('resource_rate', type=str, ...)
```

Reply:
We didn't register the arguments in the params.py file. It is not working after we call get_output_in_json method.

Reply:
Then you should register them in params.py.  This is not what transforms are intended for.